yt/visualization/plot_container.py

```diff
@@ -208,6 +216,11 @@ class PlotContainer:
     _plot_type = None
     _plot_valid = False
 
+    # Plot defaults
+    _colormap_config: dict
+    _log_config: dict
+    _units_config: dict
```

Comment:
this is more of note to myself for #3064 but 
```suggestion
    _colormap_config: Dict[FieldType: CMapType]
    _log_config: Dict[FieldType: bool]
    _units_config: Dict[FieldType: UnitType]
```